tests/pipeline/test_engine.py

```diff
@@ -510,6 +519,107 @@ def test_rolling_and_nonrolling(self):
                 ),
             )
 
+    def test_factor_with_single_output(self):
+        dates = self.dates[5:10]
+        assets = self.assets
+        num_dates = len(dates)
+        open = USEquityPricing.open
+        open_values = array([self.constants[open]] * num_dates, dtype=float)
```

Comment:
`float` here should be `float64`.  `float` will be a 32-bit float in windows, which has caused us problems in the past.